package.json

```diff
@@ -22,15 +22,15 @@
         "url": "git+https://github.com/trailheadapps/lwc-recipes"
     },
     "engines": {
-        "node": ">= 8.9.4",
-        "npm": ">= 5.7.1"
+        "node": ">= 10.13.0",
```

Comment:
I’d move this last commit to a separate PR. It’s unrelated. And the change should be discussed:
1. These are above what we’ve required for lwc itself
2. Does this create a misalignment with what sfdx and our VS Code extensions ship and require?